routes/web.php

```diff
@@ -186,20 +186,20 @@
 
 
 
-//task 3
+//task 3.1
 Route::post('users',[Xcontroller::class,'my_data']);
 
 Route::get('form',function(){
     return view('form');
 });
 
 
-// Route::post('get_data',function(Request $request){
-//     $name      =$request->input('name');
-//     $email     =$request->input('email');
-//     $message   =$request->input('msg');
-//     $subject   =$request->input('subject');
+Route::post('get_data',function(Request $request){
+    $name      =$request->input('name');
```

Comment:
here your code is correct also but there is easier syntax

for example
```php
$email     =$request->input('email');
```

more clear syntax
```php
$email     =$request->email;
```

or 
```php
$email     =$request['email'];
```